en/ui-system/components/engine/priority.md

```diff
@@ -11,3 +13,29 @@ For example:
 It can be seen from the figure that although some nodes have no renderer components, their children can participate in sorting, so these nodes also need to participate in sorting. The overall rendering order is __B -> b1 -> C -> A -> a1 -> a2__, and the rendering state on the screen is __a2 -> a1 -> A -> C -> b1 -> B__. Since we will eventually sort the nodes based on the `priority`, the order of the nodes that are ultimately presented to you is the final rendering order.
 
 For nodes that don't have a renderer component but also need to be sorted, you can use the [UIReorderComponent](../editor/ui-reorder-component.md).
+
+## 2. Mixed camera sorting
+
+The UI camera has the highest priority when it was originally designed, that is, the UI camera content is only drawn after all 3D content is drawn.However, this will cause a problem. Once the UI camera has a background or other, it will overwrite the 3D content.Therefore, a mixed sorting function between cameras is essential.
+
+The key factor of the mixed sorting of the UI camera and the 3D camera is here in the UI camera.Therefore, the `CanvasComponent` on the root node of the UI, which is the Canvas node, provides a property called __RenderMode__ to distinguish the sorting method.Next, talk about the role of the __RenderMode__ option:
+
+1. When the selection mode is __OVERLAY__, it means that the UI camera will always be behind the 3D camera,which means __will always cover the rendering content of the 3D camera__.Multiple UI cameras select this mode, and you can use the attribute __Priority__ to sort between UI cameras.
```

Comment:
1. When the selection mode is __OVERLAY__, it means that the UI camera will always be behind the 3D camera,which means __will always cover the rendering content of the 3D camera__.Multiple UI cameras select this mode, and you can use the attribute __Priority__ to sort between UI cameras.
-->
1. When the selection mode is __OVERLAY__, it means that the __UI camera__ will always be behind the __3D camera__. This means it __always covers the rendering content of the __3D camera__. When using multiple __UI cameras__ select this mode to use the attribute __Priority__ to sort between __UI cameras__.